README.rst

```diff
@@ -78,6 +78,35 @@ for more details.
 
 .. installation-end-marker
 
+Testing
+-------
+
+All code should be thoroughly tested and all pull requests should include tests.
+
+To run the Python tests, first install the package using an editable install
+as described above. The tests can then be run with
+`unittest <https://docs.python.org/3/library/unittest.html>`_.
+
+.. code-block:: bash
+
+    python -m unittest
+
+To run the C++ tests, first install the project dependencies, then setup a
+``meson`` build directory. You must configure the build as a debug build for
```

Comment:
This refers to doing `meson setup _build -Dbuildtype=debug`, correct?

Reply:
`meson setup build -Dbuildtype=debug` is what I use in the example here, but yes. Specifically the `-Dbuildtype=debug` bit.

Reply:
Oops, missed the next couple of lines here :facepalm: